File: DSA_Level-1/Basics Of Programming/2D Arrays/Search In A Sorted 2d Array/SearchInASorted2dArray_Solution.java

import java.io.*;
import java.util.*;

public class Main {

  public static void main(String[] args) throws Exception {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    int[][] nums = new int[n][n];

    for (int i = 0; i < n; ++i) {
      for (int j = 0; j < n; ++j) {
        nums[i][j] = scn.nextInt();
      }
    }
    
    int x=scn.nextInt();
    
    for (int i = 0; i < n; ++i) {
      for (int j = 0; j < n; ++j) {
        if(nums[i][j]==x){
            System.out.println(i);
            System.out.println(j);
            return;
        }
      }
    }
    
    System.out.println("Not Found");
  }

}

File: DSA_Level-1/Basics Of Programming/2D Arrays/Search In A Sorted 2d Array/SearchInASorted2dArray_Solution.js

function search(nums,x){
    for (let i = 0; i < nums.length; ++i) {
        for (let j = 0; j < nums.length; ++j) {
          if(nums[i][j]==x){
              console.log(i);
              console.log(j);
              return;
          }
        }
      }
      
      console.log("Not Found");
}

search([[11,12,13,14],[21,22,23,24],[31,32,33,34],[41,42,43,44]],51);